pandas/_libs/groupby.pyx

```diff
@@ -1420,7 +1424,7 @@ cdef cummin_max(groupby_t[:, ::1] out,
     cdef:
         Py_ssize_t i, j, N, K
         groupby_t val, mval, na_val
-        uint8_t[:, ::1] seen_na
+        uint8_t[:, ::1] seen_na = None
```

Comment:
Added this because previously `seen_na` was treated as potentially uninitialized, leading to an emission of extra code whenever it was accessed, for example
`if (unlikely(!__pyx_v_seen_na.memview)) { __Pyx_RaiseUnboundMemoryviewSliceNogil("seen_na"); __PYX_ERR(0, 1462, __pyx_L7_error)`